jsonschema/tests/test_validators.py

```diff
@@ -1662,7 +1663,7 @@ def test_False_is_not_a_schema_even_if_you_forget_to_check(self):
 
 class TestDraft3Validator(AntiDraft6LeakMixin, ValidatorTestMixin, TestCase):
     Validator = validators.Draft3Validator
-    valid = {}, {}
+    valid: typing.Tuple[dict, dict] = ({}, {})
```

Comment:
I take it mypy complains if you don't add these types here? (If not I don't see much value in adding them here for typing the test case fixtures, the real type is in fact the type of schemas, i.e. `Union[dict, bool]` right now, but possibly some larger union type later, but if it's simpler to add than ignore, then yeah leave them).

Reply:
Yes, unfortunately. `mypy` generally complains when it sees unannotated empty dict assignments.
Adding this annotation seemed the simplest way to pacify it.

We could also add `# typing: ignore` comments. For tests I think it would be appropriate (I am against the use of ignore comments in most cases, since it's just as verbose as an annotation and leaves the type checker with less information).
If we don't want type checking on tests at all, we can add `exclude = test_*` to the mypy config, and it will exclude all `test_*` module names. I could do that and revert any changes to test modules, if that is preferable.